smartseq2_single_sample/check_reference/reference_check_wf.wdl

```diff
@@ -0,0 +1,43 @@
+import "star.wdl" as RunStar
+import "featurecounts.wdl" as fc
+
+workflow CheckReference {
+
+  File input_metadata
+  Array[Object] metadata = read_objects(input_metadata)
+  String sra_dir = "gs://broad-dsde-mint-dev-teststorage/patel_ap/"
```

Comment:
I wouldn't hard-code this. It would make this really hard to reuse on different data. An alternative would be to pass the full links in 13-20 as inputs instead of constructing them within the wdl.